cmd/axon/schema.go

```diff
@@ -11,17 +14,42 @@ import (
 // maps primary key columns by table
 var primaryKeys = make(map[string][]string)
 
+func checkTargetVersion(conn *sqlx.DB) error {
+	var serverVersion string
+	err := conn.QueryRow("SHOW server_version;").Scan(&serverVersion)
+	if err != nil {
+		return err
+	}
+
+	version := strings.Split(serverVersion, ".")
+	// TODO: More thorough version checks
+	major, err := strconv.Atoi(version[0])
+	if err != nil {
+		return err
```

Comment:
I think these could be panics, or have some context added to the messages.

Reply:
Absolutely. There's a lot of UX to add. I'll add this as a todo.